### PR TITLE
[ENH, WIP] Image cropper

### PR DESCRIPTION
**Pull request by eickenberg**
Object for cropping images, addressing issue https://github.com/nilearn/nilearn/issues/198.

As of now, no exceptions if shapes and affines do not correspond. I think I should add this.

### CODE REVIEW

File: nilearn/image/image.py [hunk 1]
```diff
@@ -267,6 +269,10 @@ def crop_img(niimg, rtol=1e-8, copy=True):
         value of the image), under which values are considered
         negligeable and thus croppable.
 
+    padding: int or tuple, default 1
```
Comment (AlexandreAbraham) [inline nilearn/image/image.py]: Maybe we should explain what padding is?

**Comment by AlexandreAbraham** [comment]
@eickenberg what is the status on this PR?

**Comment by eickenberg** [comment]
does it block something? I'd definitely like to finish it, but really can't
before the beginning of July

**Comment by AlexandreAbraham** [comment]
No hurry, I'm just poking the old PRs with a stick to see if they're still alive or if we should hide the bodies :skull:

**Comment by kchawla-pi** [comment]
Is this alive? If not, please close it by this weekend.

**Pull request closed without merging**